.vscode/settings.local.recommended.json

```diff
@@ -0,0 +1,9 @@
+{
+    "_comment": [
+        "Dynamic python testing settings for the vscode testing extension."
+    ],
+    "python.testing.promptToConfigure": false,
+    "python.testing.unittestEnabled": false,
+    "python.testing.pytestEnabled": true,
+    "python.testing.pytestArgs": ["-c", "pytest.ini"]
+}
```

Comment:
For debugging and user specific settings, those can be added to `settings.local.json`. This is updated in the docs.